crates/aptos-protos/proto/aptos/block_output/v1/block_output.proto

```diff
@@ -66,6 +66,7 @@ message UserTransactionOutput {
   aptos.util.timestamp.Timestamp timestamp = 8;
   repeated SignatureOutput signatures = 9;
   string payload = 10;
+  string entry_function_id_str = 11;
```

Comment:
can this be a struct? is it a struct?

Reply:
Actually I don't need this anymore let me remove it.